frontend/src/components/Election/Voting/VotePage.tsx

```diff
@@ -10,19 +11,45 @@ import { Score } from "../../../../../domain_model/Score";
 import { Box, Container } from "@mui/material";
 import Button from "@mui/material/Button";
 const VotePage = ({ election, fetchElection }) => {
-  const { id } = useParams();
-  const [scores, setScores] = useState(election.races.map((race) =>
-    Array(race.candidates.length).fill(null)))
+  const makePages = () => {
+    // generate ballot pages
+    let pages = election.races.map((race, i) => ({
+      type: "ballot",
+      scores: Array(race.candidates.length).fill(null),
+      voting_method : race.voting_method,
+      race_index: i
+    }))
+
+    // determine where to add info pages
+    for(var i = 0; i < pages.length; i++){
+      if(pages[i].type != "ballot") continue;
+
+      // check if page is the first race with the voting method
+      var info_exists = pages.some((p) => p.type == 'info' && p.voting_method == pages[i].voting_method);
```

Comment:
We should store what info the voter has viewed in local storage so they don't see this every time.

Reply:
I'm mixed on that one. I feel like most users won't be voting very often, so they'll benefit from the fixed reminder. It could take a couple of rounds for it to fully sink in

Maybe we can add a "don't show me again" toggle and then update local storage accordingly

Reply:
Yea, a "don't show again" button would work.

Reply:
I'd like to leave this for a future PR if that's ok, I added an issue here https://github.com/Equal-Vote/star-server/issues/266

 want to test the UX without the "don't show again" button, since I think it'll be helpful for people to see even if they don't think they need it. If it get's too annoying then we can add the "do not show again" button